src/content_script/translate.ts

```diff
@@ -50,6 +51,12 @@ export async function translate(query: TranslateQuery) {
                     assistantPrompt = '翻译成简体白话文'
                 }
             }
+            if (query.selectedWords?.length > 0) {
+                // 在选择的句子中，选择特定的单词。触发语境学习功能。
+                systemPrompt = '你是一位英语词义语法专家，你在教我英语，我给你一句英语句子，和这个句子中的一个单词，请用中文帮我解释一下，这个单词在句子中的意思和句子本身的意思,如果单词在这个句子中是习话的一部分，请解释这句句子中的习话，并举几个相同意思的英语例句,并用中文解释例句。如果你明白了请说同意，然后我们开始。'
```

Comment:
Great prompt! And can English be used as the prompt when the target language is not Chinese, and explain words in the target language?

Reply:
good idea.

Reply:
> Great prompt! And can English be used as the prompt when the target language is not Chinese, and explain words in the target language?

@yetone done